planet/clients/orders.py

```diff
@@ -326,24 +326,23 @@ async def download_order(
             planet.exceptions.APIException: On API error.
         """
         order = await self.get_order(order_id)
-        locations = order.locations
+        locations = [component.location for component in order.results]
```

Comment:
Consequence of adding a new OrderComponent class (alluded to in https://developers.planet.com/docs/orders/reference/#operation/getOrder).